server/services/v1/auth/gotrue.go

```diff
@@ -687,7 +834,7 @@ func getAccessTokenUsingClientCredentialsGotrue(ctx context.Context, clientId st
 	payloadValues := url.Values{}
 	payloadValues.Set("username", clientId)
 	payloadValues.Set("password", clientSecret)
-
+	log.Error().Interface("payload", payloadValues).Msg("Payload")
```

Comment:
Is it debug message?

Reply:
yes it was left out by mistake removed it.